fiftyone/utils/cvat.py

```diff
@@ -5773,6 +5862,7 @@ def _build_frame_id_map(self, samples):
         frame_id = -1
 
         if samples.media_type == fom.VIDEO:
+            samples.ensure_frames()
```

Comment:
This is no longer necessary after my last updates in https://github.com/voxel51/fiftyone/pull/1510.